README
Framadate est un fork du projet STUdS : https://sourcesup.cru.fr/projects/studs/

Framadate est le projet qui motorise framadate.org pour framasoft.org

Les auteurs principaux de Framadate sont :
- Simon LEBLANC
- Pierre-Yves GOSSET

Les auteurs principaux du projet STUdS sont :
- Guilhem BORGHESI
- Raphaël DROZ


==========================================================================

Université de Strasbourg - Direction Informatique
Auteur : Guilhem BORGHESI
Création : Février 2008

borghesi@unistra.fr

Ce logiciel est régi par la licence CeCILL-B soumise au droit français et
respectant les principes de diffusion des logiciels libres. Vous pouvez
utiliser, modifier et/ou redistribuer ce programme sous les conditions
de la licence CeCILL-B telle que diffusée par le CEA, le CNRS et l'INRIA
sur le site "http://www.cecill.info".

Le fait que vous puissiez accéder à cet en-tête signifie que vous avez
pris connaissance de la licence CeCILL-B, et que vous en avez accepté les
termes. Vous pouvez trouver une copie de la licence dans le fichier LICENCE.

==========================================================================

Université de Strasbourg - Direction Informatique
Author : Guilhem BORGHESI
Creation : Feb 2008

borghesi@unistra.fr

This software is governed by the CeCILL-B license under French law and
abiding by the rules of distribution of free software. You can  use,
modify and/ or redistribute the software under the terms of the CeCILL-B
license as circulated by CEA, CNRS and INRIA at the following URL
"http://www.cecill.info".

The fact that you are presently reading this means that you have had
knowledge of the CeCILL-B license and that you accept its terms. You can
find a copy of this license in the file LICENSE.

==========================================================================

=============================================================================
Fichiers de l'application
=============================================================================

index.php
	La page d'accueil de STUdS
studs.php
	La page de présentation de sondage
adminstuds.php
	La page d'administration réservée à l'auteur du sondage
infos_sondage.php
	La page (1/2) de création de sondage récupérant les informations générales
choix_date.php
	La page de création (2/2) pour un sondage pour déterminer une date
choix_autre.php
	La page de création (2/2) pour un sondage sur un sujet quelconque
creation_sondage.php
	Le fichier qui récupérent les informations des pages précédentes pour procéder à l'insertion du nouveau sondage dans la base PostgreSQL
style.css
	Le fichier CSS de style pour toute l'application
variables.php
	Le fichier contenant les variables à changer en fonction de la machine locale
app/inc/functions.php
	Le fichier contenant quelques fonctions récurrentes de l'application
i18n.php
	Le fichier contenant quelques fonctions récurrentes de l'application relatives à l'internationalisation
README
	Ce fichier
INSTALL
	Le fichier contenant les informations d'installation sur l'application
CHANGELOG
	Le fichier contenant toutes les modifications de l'application entre les différentes versions
contacts.php
	La page permettant aux usagers de poser une question à l'administrateur de l'application
apropos.php
	La page expliquant les détails techniques relatifs à l'application et les dernieres modifications et celles à venir sur l'application
bandeaux.php
	Le fichier contenant tous les bandeaux des pages PHP de l'application
favicon.ico
	L'icone de favoris de l'application
sources.php
	La page qui propose les sources de l'application
exportics.php
	Le fichier d'export de la meilleure date au format iCAL (fichier .ICS)
exportcsv.php
	Le fichier d'export de tous le tableau des participants avec leurs réponses dans un tableur (format .CSV)
exportpdf.php
	Le fichier d'export de la lettre de convocation que le créateur du sondage pourra envoyer aux participants (format .PDF)

admin/
	Le répertoire réservé à l'administrateur de l'application
admin/.htaccess
	Le fichier gérant les droits restreints du répertoire ADMIN
admin/.htpasswd
	Le fichier contenant les passwd des logins ayant accès au répertoire ADMIN
admin/index.php
	La page présentant tous les sondages actuellement dans la base à l'administrateur
admin/log_studs.txt
	Le fichier contenant un historique de toutes les creations/suppressions de sondage dans la base

errors/
	Le répertoire contenant toutes les pages d'erreurs
errors/error-forbidden.php
	La page qui indique dans la charte graphique de l'application l'erreur "501 forbidden"
errors/maintenance.php
	La page qui indique que l'application est en maintenance temporaire

export/
	Le répertoire qui contient tous les exports ICS

iCalcreator/
	Le répertoire qui contient les librairies d'export en iCal

php2pdf/
	Le répertoire qui contient les librairies d'export en PDF

scripts/
	Le répertoire qui contient tous les scripts de l'application

sources/
	Le répertoire qui contient les sources de l'application disponible sur la page sources.php

locale/
	Le répertoire qui contient les fichiers de traduction modifiables (.po) et compilés (.mo)
	au format gettext

=============================================================================
	Validations des pages
=============================================================================

Toutes les pages de STUdS sont validées HTML 4.01 Strict.
La CSS de STUdS est validée CSS 2.1.

=============================================================================
Technologies utilisées
=============================================================================

- PHP 5.4.4, php-fpdf, php-adodb, php-gettext
- PostgreSQL, mysql
- Apache
- iCalcreator
- POedit
- Icônes : Deleket (http://deleket.deviantart.com/) et DryIcons (http://dryicons.com)

=============================================================================
Compatibilités des navigateurs
Dernière mise à jour le 21 avril 2014
=============================================================================

- Firefox : Ubuntu 13.10/FF28
- Chrome : Ubuntu 13.10/Chromium33
- Opera (non testé)
- Konqueror
- Links (non testé, inutile)
- Safari (non testé)
- IE : Win7/IE9

-----------------
Janvier 2008
Guilhem BORGHESI
Université de Strasbourg

Mai 2010
Raphaël DROZ, raphael.droz@gmail.com



CHANGELOG
Les dernières améliorations d'OpenSondage
    
    Changelog version 0.8 (juillet 2014 Pascal Chevrel - Armony Altinier - JosephK)
	- Améliorations sur l'accessibilité
	- Améliorations sur l'ergonomie
	- Améliorations sur l'internationalisation (nombreuses phrases en français dans le code)
	- Découpage chaines de langue pour virer le code html
	- Remise en place de l'export CSV
	- Remise en place de get_server_name() pour permettre l'installation dans un sous dossier, en https ou sur un port différent
	- Ajout Authors.md + en-têtes refaits
	- Fix bug changement de langues en mode URL rewriting (requête GET passée en formulaire POST)
	- Fix bug 2 boutons valider lorsqu'on édite un vote
	- Fix focus javascript sur "Votre nom"
	- Nettoyage + Bootstrap
	- Ajout vote Oui/Non/Si nécessaire
	- Formulaire simplifié pour l'ajout de colonne date (horaire libre)
	- Restructuration
	- Fix (partiel) bug modification du premier vote en tapant Entrée

	Changelog version 0.7 (mars 2013)
	- Fix : le sondage supprimé n'était pas forcément le sondage sélectionné (cfévrier)
	- Fix : suppression de STUDS_DIR pour éviter toute confusion
	- Fix : corrections l'en-tete et de l'encodage des e-mails (cfévrier)
	- Fix : rend Optionnelle l'utilisation de la variable "REMOTE_USER" (cfévrier)
	- Amélioration : ne faire apparaitre dans l'admin que les sondage actifs ou expirés depuis x mois (pyg)
	- Amélioration : ajout d'un champs date_creation dans la table "sondage" (pyg)
	- Amélioration : permet de faire fonctionner gettext avec le serveur de dev de PHP5.4 + enlève code commenté depuis des années (pascalchevrel)
	- Fix : enlève les appels à get_server_name() partout sauf dans un appel à sendMail(), réécriture de la fonction pour cet usage (pascalchevrel)
	- Amélioration : remplacement des define() par des const plus concis (pascalchevrel)
	- Amélioration : possibilité de faire des liens directs vers les types de sondages à créer (pascalchevrel)
	- Amélioration : meilleure intégration de la framanav (pyg)
	- Amélioration : nombreuses modifications CSS pour un meilleur affichage (pyg)
	

	Changelog des 22 et 23 juin (pyg@framasoft.net)
	- très nombreuses modifications CSS
	- ajout de buttons.css pour des boutons plus propres 
	- ajout de print.css pour une impression sans la classe "corps"
	- refonte de la page d'accueil
	- ajout de la framanav
	- qq retouches dans les fichiers .po
	- date de destruction passée de 2j à 30j
	- ajout de l'adresse à transmettre
	- ajout d'un bouton imprimer
	- généralisation des stripslashes
	- fix d'un bug sur une requete (suppression). Reste la seconde partie : https://github.com/leblanc-simon/OpenSondage/issues/8
	- modification du titre en image
	- ajout de htmlspecialchars_decode avec param ENT_QUOTES pour l'envoi des emails

	Changelog du 21 juin 2011 (pyg@framasoft.net)
	- très nombreuses modifications CSS
	- modification adminstuds.php : ajout de classes aux formulaires et ajout de stripslashes à l'affichage (TODO: à généraliser)
	- modification infos_sondages.php : simplification du tableau de choix, ajouts de CSS, ajouts de labels pour faciliter la selection
	

	Changelog version 0.6.7 (mai 2011)
	- fork du projet STUdS (https://sourcesup.cru.fr/projects/studs/) de la version trunk du 15 mai 2011)
        - reprise par Simon Leblanc
        - nettoyage du code (indentation, cohérence de la convention de codage)
        - suppression des warning php
        - résolution d'une faille de sécurité par injection SQL
        - résolution d'une faille de sécurité par injection mail
        - correction dans le fichier de langue (merci à Julien Reitzel)
        - possibilité de mettre un texte libre pour les horaires
        - version Framasoft

Les dernières améliorations de STUdS
	Changelog version 0.6.6 (XXX 2011) :
	- internationalisation avec gettext
	- abstraction de la base de données avec ADOdb
	- support de mysql (fichier d'initialisation disponible)
	- meilleure compatibilité avec le mode strict de PHP
	- factorisation de code et de CSS
	- moins de boutons de formulaire, plus de liens <a href>

	Changelog version 0.6.5 (juin 2010) :
	- Changement de deux icones dans la creation d'un sondage.

	Changelog version 0.6.4 (mars 2010) :
	- Corrections de bug

	Changelog version 0.6.3 (janvier 2010) :
	- Corrections de bug

	Changelog version 0.6.2 (novembre 2009) :
	- Correction dans l'affichage des bandeaux,
	- Modification de la partie "A propos",
	- Préparation à l'authentification,
	- De UdSification de l'application dans certains fichiers.

	Changelog version 0.6.1 (octobre 2009) :
	- Corrections d'erreurs dans les traductions et d'oublis de traduction dans certaines pages.

	Changelog version 0.6 (août 2009) :
	- Mise sous la licence CeCILL-B du code source de STUdS,
	- Passage de STUdS en encodage UTF8,
	- Ajout des icones des menus dans toutes les pages et non pas seulement sur la page d'accueil,
	- Correction d'un bug lors du rajout d'une colonne dans l'interface d'administration des sondages.

	Changelog version 0.5 (février 2009) :
	- Traduction de STUdS en anglais, allemand et espagnol,
	- Changement de la CSS avec ajout du logo de l'Université de Strasbourg,
	- Possibilité d'ajouter un commentaire pour les sondés.
	
	Changelog version 0.4 (janvier 2009) :
	- Possibilité de faire un export PDF pour envoyer la lettre de convocation à la date de réunion,
	- Possibilité de rajouter des colonnes dans la partie administration de sondage,
	- Correction de bugs d'affichage avec les caractères ' et " .

	Changelog version 0.3 (novembre 2008) :
	- Possibilité de faire un export CSV pour exploiter le sondage dans un tableur,
	- Mise en place d'un repository Subversion pour partager les nouvelles versions de STUdS,
	- Amélioration de la CSS pour un meilleur affichage,
	- Modification du code source pour le rendre portable vers une autre machine.
	
	Changelog version 0.2 (novembre 2008) :
	- Lors de la création d'un sondage DATE, classement des dates par ordre croissant,

apropos.php
<?php
/**
 * This software is governed by the CeCILL-B license. If a copy of this license
 * is not distributed with this file, you can obtain one at
 * http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.txt
 *
 * Authors of STUdS (initial project): Guilhem BORGHESI (borghesi@unistra.fr) and Raphaël DROZ
 * Authors of Framadate/OpenSondate: Framasoft (https://github.com/framasoft)
 *
 * =============================
 *
 * Ce logiciel est régi par la licence CeCILL-B. Si une copie de cette licence
 * ne se trouve pas avec ce fichier vous pouvez l'obtenir sur
 * http://www.cecill.info/licences/Licence_CeCILL-B_V1-fr.txt
 *
 * Auteurs de STUdS (projet initial) : Guilhem BORGHESI (borghesi@unistra.fr) et Raphaël DROZ
 * Auteurs de Framadate/OpenSondage : Framasoft (https://github.com/framasoft)
 */
namespace Framadate;

session_start();

include_once __DIR__ . '/app/inc/init.php';

if (file_exists('bandeaux_local.php')) {
  include_once('bandeaux_local.php');
} else {
  include_once('bandeaux.php');
}

//affichage de la page
Utils::print_header( _("About") );
bandeau_titre(_("About"));

echo <<<mentions
<ul>
  <li><a href="#faq">Questions fréquentes</a></li>
  <ul>
    <li><a href="#framadate">Qu'est-ce que Framadate
?</a></li>
    <li><a href="#studs">Quelles différences entre
Framadate et STUdS ?</a></li>
    <li><a href="#doodle">Quelles différences entre
Framadate et Doodle ?</a></li>
    <li><a href="#longevite">Mon sondage
restera-t-il longtemps en ligne ?</a></li>
  </ul>
  <li><a href="#mentions">Mentions légales</a></li>
  <li><a href="#credits">Crédits</a></li>
  <li><a href="#licence">Licence</a></li>
</ul>
<hr style="width: 100%; height: 2px;">
<h1><a name="faq"></a>Questions fréquentes</h1>
<h3><a name="framadate"></a>Qu'est-ce que
Framadate ?</h3>
Framadate est un service en ligne permettant de planifier un
rendez-vous rapidement et simplement. Aucune inscription préalable
n'est nécessaire.<br />
Framadate est un service du<a href="http://framasoft.org">
réseau Framasoft</a>, mis en place par<a
 href="http://fr.wikipedia.org/wiki/Framasoft"> l'association
Framasoft</a>.<br />
<h3><a name="studs"></a>Quelles différences
entre Framadate et STUdS ! ?</h3>
Framadate est un service basé sur le logiciel libre <a
 href="https://github.com/leblanc-simon/OpenSondage">OpenSondage</a>.
OpenSondage est lui-même basé sur le logiciel <a
 href="http://studs.u-strasbg.fr">STUdS !</a> développé
par l'Université de Strasbourg. <br />
Après avoir testé STUdS, nous avons décidé d'apporter de nombreuses
modifications, notamment ergonomiques, au code source existant.
L'ensemble de ces modifications ne pouvaient entrer dans le cadre
d'utilisation d'un logiciel déjà en production dans une université et
aurait été (fort logiquement) rejetté de la branche principale de
développement. C'est pourquoi nous avons préferer "<a
 href="http://fr.wikipedia.org/wiki/Fork_%28d%C3%A9veloppement_logiciel%29">forker</a>"
STUdS pour créer OpenSondage.<br />
<h3><a name="doodle"></a>Quelles différences
entre Framadate et <a href="http://doodle.com">Doodle</a>
?</h3>
Aujourd'hui, le danger pour le logiciel libre
ne provient plus de Microsoft ou d'Adobe et de leurs logiciels qu'on
installe sans avoir le code source, mais des applications web "dans les
nuages" proposés comme services par des entreprises.<br />
<br />
Cela pour au moins 4 raisons :<br />
1- <span style="font-weight: bold;">sécurité</span>
: aucune garantie ne peut vous être apportée quand au fait les données
soient correctement sauvegardées et protégées, ni que le<br />
code source "officiel" soit réellement celui que vous utilisez en ligne.<br />
2- <span style="font-weight: bold;">fiabilité/perennité</span>
: le service peut tomber en panne, et rien&nbsp;ne garanti que la
société Doodle sera toujours là demain et maintiendra le site<br />
3- <span style="font-weight: bold;">propriété des données</span>
: beaucoup d'entreprises s'autoproclament co-détentrices de vos
contenus "clouds" (ex: Facebook impose une clause de partage des droits
sur vos contenus, vos données, vos photos)<br />
4-<span style="font-weight: bold;"> vie privée</span>
: une entreprise - comme Doodle - doit gagner de l'argent (et il n'y a
aucun mal à cela). Mais si elle est en difficulté financière, elle peut
décider de changer ses conditions d'utilisation et vendre vos données à
des tiers (alors que Framasoft, asso loi 1901 d'intérêt général, n'aura
jamais d'intérêt à le faire).<br />
<br />
A cela s'ajoute le problème, plus éthique, de la publicité.<br />
<br />
Les problèmes 1 et 2 concernent aussi Framadate.org : rien ne garanti
la sécurité et la fiabilité du service (d'autant plus que les
administrateurs systèmes de Framasoft sont bénévoles).<br />
<br />
Par contre :<br />
- les problemes 3 et 4 ne sont pas d'actualité avec Framadate, exploité
par une association loi 1901<br />
- et surtout, Framadate fait partie d'un projet plus global
(Framatools) qui vise justement à sensibiliser le grand public à la
problématique du "cloud". Cela peut sembler paradoxal, mais bien que
proposant le service Framadate.org, nous allons surtout encourager les
organisations à installer leur propre instance du logiciel afin de
maitriser totalement leurs données.<br />
<br />
Bref, oui Framadate est inspiré de Doodle.com, et oui Doodle est un
excellent service. Mais Doodle reste une "boite noire" pour
l'utilisateur final qui va sur doodle.com. Framadate.org essaie de
répondre, modestement, à cette problématique en montrant que des
alternatives libres existent et qu'on peut les installer "chez soi".<br />
<h3><a name="longevite"></a>Mon sondage
restera-t-il longtemps en ligne ?</h3>
Le service Framadate est proposée gratuitement par l'association
Framasoft.<br />
Framasoft
s'engage à maintenir le service "aussi longtemps que possible", mais ne
peut fournir aucune garantie de date. Si cela ne vous convient pas,
nous vous
encourageons sincèrement à installer vous-même Framadate et à maintenir
vous-même votre propre service.
<hr style="width: 100%; height: 2px;">
<h1><a name="mentions"></a>Mentions légales<br />
</h1>
<h2>Éditeur et Responsable de la publication</h2>
<p>Editeur : Association Framasoft (cf "Hébergement")</p>
<p>Responsable de la publication : Christophe Masutti</p>
<p>Vous pouvez rentrer en contact avec l'Editeur et le
Responsable de la publication en passant par la page "<a
 href="http://contact.framasoft.org">contact</a>".</p>
<p>Les propos tenus sur ce site ne représentent que et uniquement
l’opinion de leur auteur, et n’engagent pas l'association Framasoft,
les sociétés, entreprises ou collectifs auxquels il contribue ou dont
il peut être associé ou employé.</p>
<h2>Hébergement</h2>
<p>Ce site est hébergé par Framasoft, 10 bis rue Jangot 69007 Lyon, France.
Cet hébergeur possède à ce jour les éléments d’identification
personnelle concernant l'Éditeur (voir <a
 href="http://www.framasoft.net/article4736.html">http://www.framasoft.net/article4736.html</a>).</p>
<h2>Données personnelles</h2>
<p>Les données personnelles collectées par Framadate sont
uniquement destinées à un usage interne. En aucun cas ces données ne
seront cédées ou vendues à des tiers.
Conformément à l’article 39 de la loi du 6 janvier 1978 relative à
l’informatique, aux fichiers et aux libertés, vous avez un droit
d’accès, de modification et d’opposition sur vos données personnelles
enregistrées par le blog.
Dans ce cas, utilisez le formulaire de contact.</p>
<h2>Conditions de modération/suppression de sondages</h2>
<p>Les sondages de Framadate bénéficient d'une URL aléatoire,
mais publique. Si vous souhaitez supprimer un sondage, utilisez
l'adresse d'aministration fournie à la création. Vous pouvez
exceptionnellement demander la suppression d'un sondage en utilisant la
page de contact.</p>
<h2>Notification des contenus litigieux</h2>
<p>Conformément à l’article 6 I 5° LCEN, la connaissance des
contenus litigieux est présumée acquise par L’Éditeur lorsqu’il lui est
notifié, par lettre recommandée avec accusé de réception adressée au
siège social de L’Éditeur, la totalité des éléments suivants (i) la
date de la notification&nbsp;; (ii) si le notifiant est une
personne physique&nbsp;: ses nom, prénoms, profession, domicile,
nationalité, date et lieu de naissance&nbsp;; si le notifiant est
une personne morale&nbsp;: sa forme, sa dénomination, son siège
social et l’organe qui la représente légalement&nbsp;; (iii) les
nom et domicile du destinataire ou, s’il s’agit d’une personne morale,
sa dénomination et son siège social&nbsp;; (iv) la description des
faits litigieux et leur localisation précise&nbsp;; (v) les motifs
pour lesquels le contenu doit être retiré, comprenant la mention des
dispositions légales et des justifications de faits&nbsp;; (vi) la
copie de la correspondance adressée à l’auteur ou à l’éditeur des
informations ou activités litigieuses demandant leur interruption, leur
retrait ou leur modification, ou la justification de ce que l’auteur ou
l’éditeur n’a pu être contacté.</p>
<p>A défaut d’envoi de la totalité de ces éléments, la
notification ne sera pas prise en compte par L’Éditeur et ce dernier ne
pourra en conséquence être présumé informé d’un contenu litigieux.</p>
<p>L’Éditeur se réserve le droit d’engager des poursuites à
l’encontre de toute personne ayant usé abusivement du droit réservé par
l’article 6 I 4° LCEN. L’Éditeur vous rappelle que toute personne qui
aurait présenté un contenu ou une activité comme étant illicite dans le
but d’en obtenir le retrait ou d’en faire cesser la diffusion alors
qu’elle a connaissance du caractère inexact de cette information, est
susceptible d’encourir une peine d’un an d’emprisonnement et de 15.000
€uros d’amende.</p>
<h2>Licences, droits de reproduction</h2>
<p>L'application Framadate, basé sur le logiciel OpenSondage,
lui-même basé sur STUdS, est publiée sous licence libre <a
 href="http://www.cecill.info/licences.fr.html">CeCILL-B</a>.
Les contenus (sondages) sont publiés sous licence Creative Commons
BY-SA. Cela signifie que si l'adresse de votre sondage est connue d'un
individu, vous autorisez cette personne à utiliser, partager, modifier
votre sondage. Si vous souhaitez des sondages 100% privés et avec votre
propre licence, installez votre propre logiciel de sondage et
n'utilisez pas Framadate.org.</p>
<hr style="width: 100%; height: 2px;">
<h2><a name="credits"></a>Crédits</h2>
<b>Application d'origine</b><br />
<br />
L'application Framadate est une instance du logiciel <b><a
 href="http://studs.u-strasbg.fr">STUdS !</a></b>
développé à l'Université de Strasbourg depuis 2008.<br />
<br />
Pour les besoins de Framadate, STUdS a fait l'objet d'un fork par
l'équipe Framasoft. Les sources sont disponibles sur le Github <a
 href="https://github.com/leblanc-simon/OpenSondage">OpenSondage</a>.<br />
<br />
<b>Technologies utilisées</b><br />
<br />
- <a href="http://www.php.net/">PHP</a><br />
- <a href="http://www.postgresql.org/">MySQL</a><br />
- <a href="http://www.apache.org/">Apache</a><br />
- <a href="http://subversion.tigris.org/">Subversion</a><br />
- <a href="http://www.kigkonsult.se/iCalcreator/">iCalcreator</a><br />
- <a href="http://www.fpdf.org/">FPDF</a><br />
- Icônes : <a href="http://deleket.deviantart.com/">Deleket</a>,
<a href="http://pixel-mixer.com">PixelMixer</a> et <a
 href="http://dryicons.com">DryIcons</a><br />
<br />
<b>Compatibilités des navigateurs</b><br />
<br />
- <a href="http://www.mozilla.com/firefox/">Firefox</a><br />
- <a href="http://www.opera.com/">Opéra</a><br />
- <a href="http://www.konqueror.org/">Konqueror</a><br />
- <a href="http://www.jikos.cz/%7Emikulas/links/">Links</a><br />
- <a href="http://www.apple.com/fr/safari/">Safari</a><br />
- <a href="http://www.mozilla.com/firefox/">IE</a><br />
<br />
<b>Validations des pages</b><br />
<br />
- Toutes les pages disposent de la validation HTML 4.01 Strict du W3C. <br />
- La CSS dispose de la validation CSS 2.1 du W3C.
<p><img src="http://www.w3.org/Icons/valid-html401-blue"
 alt="Valid HTML 4.01 Strict" height="31" width="88"><img
 style="border: 0pt none ; width: 88px; height: 31px;"
 src="http://jigsaw.w3.org/css-validator/images/vcss-blue"
 alt="CSS Valide !">
</p>
<b>Propositions améliorations de Framadate</b><br />
<br />
Si quelque chose venait à vous manquer, vous pouvez nous en faire part
via le <a href="http://contact.framasoft.org">formulaire en ligne</a>.
<br />
Les dernières améliorations de Framadate sont visibles dans le fichier <a
 href="CHANGELOG">CHANGELOG</a>.<br />
<br />
<b>Remerciements</b><br />
<br />
<b><a href="http://studs.u-strasbg.fr">STUdS !</a></b>&nbsp;:
Pour leurs contributions techniques ou ergonomiques : Guy, Christophe,
Julien, Pierre, Romaric, Matthieu, Catherine, Christine, Olivier,
Emmanuel et Florence <br />
<a style="font-weight: bold;"
 href="https://github.com/leblanc-simon/OpenSondage">Framadate</a>
: &nbsp;Simon Leblanc (développement principal), Pierre-Yves Gosset
(développement, graphisme), la communauté Framasoft.<br />
<br />
<h2><b><a name="licence"></a>Licence</b></h2>
Framadate est placé, comme <b><a
 href="http://studs.u-strasbg.fr">STUdS !</a>,</b>
sous la licence logicielle libre <a
 href="http://www.cecill.info/licences.fr.html">CeCILL-B</a>.<br />
<br />
mentions;

bandeau_pied();
